apps/server/src/services/project-service/ProjectService.ts

```diff
@@ -89,17 +94,29 @@ export async function getProjectFiles(): Promise<ProjectFile[]> {
   return projectFiles;
 }
 
+export async function getLoadedProject(): Promise<string> {
+  const appState = await appStateService.get();
+
+  if (appState.lastLoadedProject) {
+    const filePath = join(resolveProjectsDirectory, appState.lastLoadedProject);
+    if (!checkIfFileExists(filePath)) {
+      await appStateService.updateDatabaseConfig('');
+      return '';
+    }
+  }
+  return appState.lastLoadedProject;
```

Comment:
question: i wonder if we can stop it from happening here. 
Checking for loaded project should happen a) when a project is loaded b) when the app is started

From there we just get the state from the appStateService.
I dont see a good scenario for receiving an empty string here